vscode/src/client/analyzerClient.ts

```diff
@@ -276,11 +281,11 @@ export class AnalyzerClient {
         },
       );
 
-      console.log("response", response, incident, state);
-      vscode.commands.executeCommand("konveyor.loadSolution", response, { incident, violation });
+      vscode.commands.executeCommand("konveyor.loadSolution", response, {
```

Comment:
it seems we have different formatting settings in the IDE

Reply:
Using prettier on my end. Not sure if there is a default prettier config that comes with the prettier plugin that may differ.

Reply:
the line length is configured in the `.editorconfig` 
https://github.com/konveyor/editor-extensions/blob/9e3c6f3196a9fbb703124c57d403cd5ff15bd5bf/.editorconfig#L13